docs/tutorials/working_with_the_ensemble.ipynb

```diff
@@ -19,33 +19,42 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    # Pass the ColumnMapper along to from_source_dict
</pre><p>update comment to from_pandas</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tape/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Done.</p>